source/includes/_tenants.md

```diff
@@ -0,0 +1,269 @@
+# Tenants
+
+## The Tenant Resource
+
+<aside class="notice">
+This section describes API features that are part of the Cloudify premium edition
+</aside>
+
+> `Note`
+
+```python
+# include this code when using cloudify python client-
+from cloudify_rest_client import CloudifyClient
+client = CloudifyClient(
+        host='<manager-ip>',
+        username='<manager-username>',
+        password='<manager-password>',
+        tenant='<manager-tenant>')
+```
+
+```html
+
+```
+
+The Tenant resource is a logical component that represent a closed environment with its oun resources.
```

Comment:
The Tenant resource is a logical component that **represents** a closed environment with its ~~oun~~ **own** resources.